### ISSUE TITLE
Still supporting this?

### ISSUE THREAD

**Issue by siredwin** [bamzi/jobrunner#34]
First of all, thanks for this. It helped me build my own cron job manager.
I would still like to use jobrunner for regular internal crons.
Are you still supporting it?

**Comment by bamzi** [comment]
Please note this lib is planned for deprecation or archiving. The reason is simple, when I created this I was naive and wanted to do a lot on http API requests. However, after upgrading my knowledge to event driven architectures and signals, the use of this kind libraries are not really recommended

**Comment by siredwin** [comment]
I agree with you on EDA. There are better libraries for responding to events. I think this falls in to the class of cron monitors. There are situations where you need to trigger events based on time. This is a very good basic example. Consider changing the description and archive it. I was able to create my own up-time web monitor based on this.
